contracts/starknet/objects/proposal.cairo

```diff
@@ -0,0 +1,6 @@
+struct Proposal:
+    member execution_hash : felt  # TODO: Use Hash type
+    member metadata_uri : felt  # TODO: Use String type
+    member start_block : felt
```

Comment:
Did you tried with timestamp here?

Reply:
https://github.com/snapshot-labs/sx-core/pull/47/commits/ce41532f7ea1496460fdde71f80cb5e24d558a25